pkg/config/system_test.go

```diff
@@ -567,7 +567,7 @@ func TestGetZoneConfigForKey(t *testing.T) {
 		{roachpb.RKey(keys.TimeseriesPrefix.PrefixEnd()), keys.SystemRangesID},
 		{roachpb.RKey(keys.TableDataMin), keys.SystemDatabaseID},
 		{roachpb.RKey(keys.SystemConfigSplitKey), keys.SystemDatabaseID},
-		{roachpb.RKey(keys.GracePeriodInitTimestamp), keys.SystemRangesID},
+		{roachpb.RKey(keys.ClusterInitGracePeriodTimestamp), keys.SystemRangesID},
```

Comment:
I assume changing this is safe in the backport.

Reply:
Great point. Yes, it’s safe—this is just a variable rename, and the actual key stored on disk remains unchanged. Plus, we aren’t using the key yet since the license enforcer starts disabled, so it’s doubly safe.